ckan_meta_tester/__init__.py

```diff
@@ -10,10 +13,26 @@ def test_metadata() -> None:
     logging.getLogger('').setLevel(
         environ.get('INPUT_LOG_LEVEL', 'info').upper())
 
+    github_token = environ.get('GITHUB_TOKEN')
+
+    # Get PR body text
+    pr_url = environ.get('INPUT_PULL_REQUEST_URL')
+    pr_body = ''
+    if pr_url:
+        headers = { 'Accept': 'application/vnd.github.3.raw+json' }
+        parsed_pr_url = urlparse(pr_url)
+        if github_token:
+            if parsed_pr_url.scheme == 'https' and parsed_pr_url.netloc == 'api.github.com' \
+                    and parsed_pr_url.path.startswith('/repos/'):
+                headers['Authorization'] = f'Bearer {github_token}'
+            else:
+                logging.warning('Invalid pull request url, omitting Authorization header')
+        pr_body = requests.get(pr_url, headers=headers).json().get('body', '')
```

Comment:
This is probably enough self-contained code now to split out into a new function, what do you think?